cmd/access/main.go

```diff
@@ -125,7 +145,7 @@ func main() {
 			)
 			return err
 		}).
-		Module("collection node client", func(node *cmd.FlowNodeBuilder) error {
+		Module("collection node client", func(builder cmd.NodeBuilder, node *cmd.NodeConfig) error {
```

Comment:
@vishalchangrani Why are we passing in both `NodeBuilder` and `NodeConfig`? It looks like the builder struct embeds the config.